app/scripts/lib/rpc-method-middleware/handlers/add-ethereum-chain.js

```diff
@@ -158,15 +158,16 @@ async function addEthereumChainHandler(
     // If this network is already added with but is not the currently selected network
     // Ask the user to switch the network
     try {
-      await updateRpcTarget(
+      await setNetworkTarget(
         await requestUserApproval({
           origin,
           type: MESSAGE_TYPE.SWITCH_ETHEREUM_CHAIN,
           requestData: {
             rpcUrl: existingNetwork.rpcUrl,
             chainId: existingNetwork.chainId,
-            nickname: existingNetwork.nickname,
+            chainName: existingNetwork.chainName,
             ticker: existingNetwork.ticker,
+            uuid: existingNetwork.uuid,
```

Comment:
Since this property refers to the network configuration, should this be `networkConfigurationId` rather than `uuid`?

Reply:
done in latest!